webpack.config.demo.js

```diff
@@ -31,7 +31,9 @@ module.exports = [{
             loader: 'css-loader',
             options: {
               modules: {
-                localIdentName: '[name]__[local]__[hash:base64:5]',
+                localIdentName: process.argv.includes('production')
```

Comment:
This seems a bit brittle.
I would prefer to detect the mode explicitly. Something like this:
```
...
module.exports = (env, options) => [{`
...
localIdentName: options.mode === 'production'
                  ? '[hash:base64:8]'
                  : '[name]__[local]__[hash:base64:8]',
...
```

Same in `webpack.config.lib.js`